setup.py

```diff
@@ -74,5 +74,10 @@
             'dev': test_requirements + doc_requirements,
         },
 
+        entry_points="""
+                [console_scripts]
+                deaths_model=covid_model_deaths.cli.cli:deaths_model
+            """,
+
```

Comment:
My package entry points are a data structure and not a string. Consider using a structure

```
entry_points={'console_scripts': [
        'run_extract = winnower.commands.run_extract:run_extract',
        'generate-extraction-hook-template = winnower.commands.generate_extraction_hook_template:entry_point',
        'list_extraction_ids = winnower.commands.list_extraction_ids:list_extraction_ids',
        'winnower_fix_spss = winnower.commands.winnower_macosx_dyld_path:main',
    ]},
```